client/src/components/csv/CSVReview.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import {Container, Label, Item} from 'semantic-ui-react';
+import { ItemComplete } from "../item/ItemEdit";
+
+interface ReviewCardProps {
+    item: ItemComplete
+}
+
+interface ReviewProps {
+    inventory: ItemComplete[]
+}
+
+const ReviewSetup = (props: ReviewProps) => {
+    const { inventory } = props;
+
+    return (
+        <Container>
+            <Item.Group>
+                {inventory.map(item => <ReviewCard key={item.name} item={item}/>)}
+            </Item.Group>
+        </Container>
+    );
+}
+
+const ReviewCard = (props: ReviewCardProps) => {
+    const { item } = props;
+    const { name, description, totalQty, maxReqQty, 
+            imageUrl, category, price, owner, 
+            requireApproval, returnRequired, hidden} = item;
+    return (
+        <Item>
+            <Item.Image size='tiny' src={imageUrl}/>
+            <Item.Content>
+                <Item.Header as='h4'>{name}</Item.Header>
+                <Item.Meta>{maxReqQty} requests limit | {totalQty} available, Owner: {owner}, Unit Cost: {price}</Item.Meta>
+                <Item.Meta>
+                    <Label>{category}</Label>
+                    {hidden ? <Label tag color='orange'>Hidden</Label>: null}
+                    {requireApproval ? <Label tag color='red'>Requires Approval</Label>: null}
```

Comment:
Small semantics thing here, not sure of my final feelings about this but IMO items requiring approval and return is the default and is not out of the ordinary.  So maybe these labels should call out when items do **not** require approval or do **not** require return, since those might be mistakes.